support-frontend/assets/components/content/content.scss

```diff
@@ -16,14 +16,6 @@
 	@if ($text) {
 		color: $text;
 	}
-	@if ($bg) {
-		&,
-		.component-content-bg {
```

Comment:
I thought it made sense to remove this at the same time as the export, just in case we port it over to Emotion to understand what was there to start with